Build.mak

```diff
@@ -26,6 +26,12 @@ TEST_FILTER_OUT += \
 TEST_FILTER_OUT += \
 	$C/test/collectd/main.d 
 
+# integration tests which are temporarily disabled due to flakiness
+# to be fixed and re-enabled
+TEST_FILTER_OUT += \
+	$C/test/unixsocket/main.d \
```

Comment:
I suspect `unixlistener` is flaky as well (just it happened it didn't already failed), as it is using the same best-effort approach to synchronizing processes.